src/SparseVariationalApproximationModule.jl

```diff
@@ -306,7 +306,7 @@ Statistics. PMLR, 2015.
 """
 function AbstractGPs.elbo(
     sva::SparseVariationalApproximation,
-    fx::FiniteGP{<:AbstractGP,<:AbstractVector,<:Diagonal{<:Real,<:Fill}},
+    fx::FiniteGP{<:AbstractGP,<:AbstractVector,<:Union{Diagonal{<:Real,<:Fill},ScalMat}},
```

Comment:
This should probably be in a seperate issue, but I think that this is too restrictive of a type constraint - it depends too much on implementation details of AbstractGPs (which is why it broke in the first place) and isn't extensible. I'm pretty sure I wrote it in the first place, so I'll make a PR to change it once this one's in - but what do people think about it?

Reply:
I'd be happy for it to become cleaner, go ahead:) - what would be a better way of checking though?